### ISSUE TITLE
Chunks dissappear randomly, and when players do /suicide

### ISSUE THREAD

**Issue by Momo99-hub** [Imanity-Software/ImanitySpigot3-git#53]
### Stack trace

```
paste your stack trace or a https://paste.imanity.dev/ link here!
```

https://paste.imanity.dev/ABB8MeU

### Plugin list

Check paste bin.

### Steps/models to reproduce (if known)

I cant reproduce but it just happens after a while when players play, randomly chunks will just dissappear and will come back on restart or just after a while... also all chunks dissappear when players do /suicide

### ImanitySpigot3 version

LATEST

### Agreements

- [X] I am running the latest version of ImanitySpigot3 available from
- [X] I have searched for and ensured there isn't already an open issue regarding this.
- [X] I have compared the issue inbetween TacoSpigot 1.8.8 and ImanitySpigot3 And only ImanitySpigot3 occured this issue.

**Comment by Momo99-hub** [comment]
It happend right before I restarted the server. As you can see there is no sign of this happening...

**Comment by LeeGodSRC** [comment]
Can you try to reproduce the bug without any plugins?

**Comment by Momo99-hub** [comment]
Yes, and I only had this problem with iSpigot 3, it used to happen way more frequently but it got better, but it still happens.

**Comment by LeeGodSRC** [comment]
We wasn't able to reproduce the problem
and the information you provided wasn't enough for the case
more details are needed

1. in the ImanitySpigot3 version box you said LATEST but **we clearly stated do not say LATEST, as it could be misleading. please change it to an actual version you are using**. (take a picture from /version is enough)
2. Can you send us a video to tell where the problem occurs?
3. Does the problem still occurs when you disabled "asyncChunkLoads.enabled" in imanity/config.yml?
4. Please send us the entire server setup (include everything like configuration / worlds etc) that you were able to reproduce the bug **in ticket** so we can test out the problem in our end.

**Comment by LeeGodSRC** [comment]
No response within last 5 days, please re-open whenever you receive new information.

**Comment by AlexCanaryYT** [comment]
I can vouch this happens, it does it randomly when you restart the server. No possible to reproduce since its random. I have "asyncChunkLoads" disabled completely.

**Comment by LeeGodSRC** [comment]
> We wasn't able to reproduce the problem and the information you provided wasn't enough for the case more details are needed
> 
> 1. in the ImanitySpigot3 version box you said LATEST but **we clearly stated do not say LATEST, as it could be misleading. please change it to an actual version you are using**. (take a picture from /version is enough)
> 2. Can you send us a video to tell where the problem occurs?
> 3. Does the problem still occurs when you disabled "asyncChunkLoads.enabled" in imanity/config.yml?
> 4. Please send us the entire server setup (include everything like configuration / worlds etc) that you were able to reproduce the bug **in ticket** so we can test out the problem in our end.

@AlexCanaryYT will you be able to provide 3 other information I asks above? It would be helpful.

**Comment by AlexCanaryYT** [comment]
> > We wasn't able to reproduce the problem and the information you provided wasn't enough for the case more details are needed
> > 
> > 1. in the ImanitySpigot3 version box you said LATEST but **we clearly stated do not say LATEST, as it could be misleading. please change it to an actual version you are using**. (take a picture from /version is enough)
> > 2. Can you send us a video to tell where the problem occurs?
> > 3. Does the problem still occurs when you disabled "asyncChunkLoads.enabled" in imanity/config.yml?
> > 4. Please send us the entire server setup (include everything like configuration / worlds etc) that you were able to reproduce the bug **in ticket** so we can test out the problem in our end.
> 
> @AlexCanaryYT will you be able to provide 3 other information I asks above? It would be helpful.

1.
![image](https://user-images.githubusercontent.com/74209461/141659828-e89ef166-b474-43e2-a52c-b566fa845444.png)

2. I do not have a video right now since it happens randomly, but basicly its just that the chunk does not load and then you get into the void for a couple of seconds until it loads, but most of our anticheats false flag due to that problem and many players get banned after that.
3. Yes, the problem still happens after disabling that.
4. I dont understand exactly what sending the entire setup is but im not gonna send anything since I've already explained all.

**Comment by LeeGodSRC** [comment]
This issue has been solved in **2021.11.1 LTS BUILD 5**
